### ISSUE TITLE
Windows binary release of 1.1.0 missing

### ISSUE THREAD

**Issue by djcater** [nabla-c0d3/sslyze#210]
Hello,

The README (https://github.com/nabla-c0d3/sslyze#windows-executable) says:

> A pre-compiled Windows executable is available in the Releases tab

And indeed previous releases (such as 1.0.0) have a downloadable pre-compiled Windows binary on the releases page: https://github.com/nabla-c0d3/sslyze/releases

Please can one also be added for the recent 1.1.0 release?

Thank you.

**Comment by nabla-c0d3** [comment]
I will add it back once I add support for Python 3 on Windows.

**Comment by nottheend** [comment]
Tried to build it as described in README.md: `python.exe setup_py2exe.py py2exe`
It is not successful, receiving the following error:
```
File "sslyze\setup_py2exe.py", line 7, in <module>
    import py2exe
ModuleNotFoundError: No module named 'py2exe'
 ```

**Comment by nottheend** [comment]
Was able to install sslyze on Windows only with Python 2.7, enviroment managed by Anaconda

**Comment by nabla-c0d3** [comment]
This is fixed in 1.1.3 https://github.com/nabla-c0d3/sslyze/releases/tag/1.1.3
